rts/Rendering/Env/Particles/Classes/ShieldSegmentProjectile.cpp

```diff
@@ -43,7 +43,7 @@ static spring::unsynced_map<const AtlasedTexture*, std::vector<float2> > spheret
 
 
 
-ShieldSegmentCollection& ShieldSegmentCollection::operator = (ShieldSegmentCollection&& ssc) {
+ShieldSegmentCollection& ShieldSegmentCollection::operator=(ShieldSegmentCollection&& ssc) noexcept {
```

Comment:
a "spaces versus no spaces for overloaded operators" rule was never established, this could be a long road

Reply:
Hm, agreed, code seems mixed on that. Think I'll avoid doing any (extra) changes.